src/haxe/crypto/Sha224.hx

```diff
@@ -27,9 +27,9 @@ import haxe.ds.Vector;
 */
 class Sha224 {
 
-    public static function encode( s:String, ?encoding : haxe.io.Encoding ) : String {
+    public static function encode( s:String #if !haxe3 , ?encoding : haxe.io.Encoding #end ) : String {
         var sh = new Sha224();
-		var data = haxe.io.Bytes.ofString(s, encoding);
+		var data = haxe.io.Bytes.ofString(s #if !haxe3 , encoding #end );
```

Comment:
I still don't think this works, given that `haxe3` is defined for both Haxe 3 and 4? `#if haxe4` should be better?

Reply:
Yes indeed. Thanks